libraries/00_LIBS/src/lsm6dsm.cpp

```diff
@@ -549,7 +549,7 @@ float LSM6DSM::calcAccel(int16_t input)
 /**
   * @brief  Read gyro Z-axis raw data   
   * @param  None
-  * @retval X-axis raw data
+  * @retval Z-axis raw data
```

Comment:
This should be left as it was, the one on the 550th line should be changed to X-axis from Z-axis.